charts/templates/statefulset.yaml

```diff
@@ -43,8 +43,8 @@ spec:
               log "Error: [initContainer] Bootstrap host $MARKLOGIC_BOOTSTRAP_HOST not found, exiting Init container."
               exit 1
             fi
-            GROUP_CFG_TEMPLATE='{"group-name":"%s"}'
-            GROUP_CFG=$(printf "$GROUP_CFG_TEMPLATE" "$MARKLOGIC_GROUP")
+            GROUP_CFG_TEMPLATE='{"group-name":"%s", "xdqp-ssl-enabled":"%s"}'
```

Comment:
This part of logic looks good. However, this only covers for the case when creating a new group. You might have missed the logic to config the XDQP encryption when it is the first cluster, in which case you need to add the same logic to postStart hook.